packages/forms-web-app/src/controllers/lpa-dashboard/appeal-details.js

```diff
@@ -26,6 +26,16 @@ const getAppealDetails = async (req, res) => {
 		caseReference,
 		'Appeal%20Statement'
 	);
+	const supportingDocumentsMetaData = await getAppealDocumentMetaData(
+		caseReference,
+		'Supporting%20Documents',
```

Comment:
The only other thing I thought of was doing something like https://www.geeksforgeeks.org/node-js-querystring-encode-function/
querystring.encode('Supporting Documents'). Maybe we've tried this and it doesn't work though, I know we had some discussions about this stuff.
It could also be a constant rather than inline string.